### ISSUE TITLE
Elite sports team selector

### ISSUE THREAD

**Issue by Tamaralb** [alptium/tamara.sevic#6]

**Comment by Tamaralb** [comment]
@valentinacupac  I'm finished, you can check.

**Comment by valentinacupac** [comment]
@Tamaralb, nice solution, here are some points to improve:
1. There is both Pass and Player, they seem to be duplicates
2. Why do we have @SuppressWarnings("null") because in general we should avoid suppressing warnings, unless it's really really required
3. Fix indentation and formatting, see https://github.com/alptium/valentina.cupac/wiki/03.-Hello-World and maybe see code examples from Oracle java Tutorials.
4. Will the program count be correct?
5. Regarding the multiple conditions which caused rejection, they are currently in multiple if-else-if-else-if-else-etc statements, is it possible that they could have been combined together and be just if-else? Let me know if possible?
6. Please also here inside the ticket, write the logic you implemented, but write it in plain English, so that I can see if the code corresponds to the logic.
7. Why is listPass outside the method, please check...

**Comment by Tamaralb** [comment]
@valentinacupac  I think it's now right.  The idea is that candidates enter their data and that based on the criteria, that i found on the internet related to water polo, will be accepted or not. It also has the possibility for entry more candidates at the same time of different sexes and finally prints the list of candidates.

**Comment by valentinacupac** [comment]
@Tamaralb, 

1. Ok.
2. Ok.
3. The formatting is still not fixed, e.g. the nested if-s are for example not correctly formatted and there's double blank lines.
4. The usage of the word "count" here "int count" is not really correct, because what if the list is empty but you put count = 1, so that's not conceptually correct. Could you instead maybe use a word like int candidateNumber?
5. Ok.
6. Expalin to me lines 41-60 as plain English statements, i.e. imagine you are writing requirements spec to explain to someone what the selection logic is.
7. Ok.
8. Ok.
9. Why do you have mixed English and Serbian, e.g. line 100?

**Comment by Tamaralb** [comment]
@valentinacupac 6. I have different criteria for men and women. If a candidate meet all the conditions, I add it to the list of candidates who have passed.

**Comment by valentinacupac** [comment]
@Tamaralb:
3. Formatting was still not correct, see the push from me on your code now. Let me know if you understood it, or need any clarification?
10. I would also like to see at the end, not just the candidates who passed but also the candidates who failed, and also statistics total who applied, total accepted, total rejected.

**Comment by Tamaralb** [comment]
@valentinacupac I think that now it is right, you can check.
